test/FsAutoComplete.Tests.Lsp/Tests.fs

```diff
@@ -502,15 +502,38 @@ let fsdnTest () =
       }
     ])
 
+let dotnetnewlistTest () =
+  let path = Path.Combine(__SOURCE_DIRECTORY__, "TestCases", "Empty")
+  serverTest path defaultConfigDto (fun (server, event) ->
+    do waitForWorkspaceFinishedParsing event
+
+    testList "DotnetNewList Tests" [
+      testAsync "DotnetNewList on list" {
+        let p : DotnetNewListRequest = {
+            Query = "Console Application"
+        }
+
+        let! res = server.FSharpDotnetNewList p
+        match res with
+        | Result.Error e -> failtestf "Request failed: %A" e
+        | Result.Ok n ->
+          Expect.stringContains n.Content "Console Application" (sprintf "the Console Application is a valid response, but was %A" n)
+          let r = JsonSerializer.readJson<CommandResponse.ResponseMsg<CommandResponse.DotnetNewListResponse>>(n.Content)
+          Expect.equal r.Kind "dotnetnewlist" (sprintf "dotnetnewlist response, but was %A, from json %A" r n)
+          Expect.contains r.Data.Functions "Console Application" (sprintf "the Console Application is a valid response, but was %A, from json %A" r n)
```

Comment:
the build is failing to compile on this line.
see https://ci.appveyor.com/project/fsautocomplete/fsautocomplete/builds/25614351

the error is 

```
C:\projects\fsautocomplete\test\FsAutoComplete.Tests.Lsp\Tests.fs(523,34): error FS0039: The field, constructor or member 'Functions' is not defined. [C:\projects\fsautocomplete\test\FsAutoComplete.Tests.Lsp\FsAutoComplete.Tests.Lsp.fsproj]
```

does `CommandResponse.DotnetNewListResponse` contains a field `Functions`?

Reply:
Yes, I have looked into the appveyor build error. The `CommandResponse.DotnetNewListResponse` contains field `Installed` not `Functions`.
I'm currently working on to solve the error.